packaging/opensuse-42.2/snapd.spec

```diff
@@ -293,6 +298,7 @@ fi
 %{_mandir}/man5/snap-discard-ns.5.gz
 %{_unitdir}/snapd.service
 %{_unitdir}/snapd.socket
+%{_unitdir}/snapd.seeded.service
```

Comment:
NOTE: I had to add this or running spread manually would fail with unpackaged files. Probably a fallout from the change above where we install additional systemd units.

CC @mvo5